### ISSUE TITLE
Unable to Login

### ISSUE THREAD

**Issue by Roxville** [MscrmTools/XrmToolBox#301]
Downloaded newest version.  Issue that I was having with the older version are gone (post on old site)...however, for whatever reason the login fails regardless of using internal or external link:

http://dynamics.*******.local:5555
https://dynamics.*******.com:5556 (IFD)

Both links work to authenticate using IE and Outlook and authenticate properly.  My user account is a system administrator within Dynamics.

Local login has domain, user, password.
IFD used both domain\username and username@domain with password.

Not sure what else you might need...other than this is an on premise install of Dynamics CRM 2015.

**Comment by MscrmTools** [comment]
This looks like a specific configuration with some unsupported stuff... It is not allowed to have two bindings for CRM website
Can you connect with plugin registration tool?

**Comment by Roxville** [comment]
Hmmm, Microsoft's documentation never says to remove the initial binding.  Will try without the 5555; the configuration is step by step from Microsoft, not sure what would be unsupported...

Where do I find the plugin registration tool?

**Comment by Roxville** [comment]
Had to find the 2015 SDK, but here is the trace:

Source	: Microsoft.Xrm.Sdk
Method	: GetIssuerEndpoint
Date	: 12/22/2015
Time	: 4:10:26 PM
Error	: The authentication endpoint Username was not found on the configured Secure Token Service!
Stack Trace	: at Microsoft.Xrm.Sdk.Client.IssuerEndpointDictionary.GetIssuerEndpoint(TokenServiceCredentialType credentialType)
   at Microsoft.Xrm.Sdk.Client.ServiceConfiguration`1.AuthenticateInternal(AuthenticationCredentials authenticationCredentials)
   at Microsoft.Xrm.Sdk.Client.ServiceConfiguration`1.AuthenticateFederationInternal(AuthenticationCredentials authenticationCredentials)
   at Microsoft.Xrm.Sdk.Client.ServiceConfiguration`1.Authenticate(AuthenticationCredentials authenticationCredentials)
   at Microsoft.Xrm.Tooling.Connector.CrmWebSvc.ClaimsIFDFailOverAuth[T](IServiceManagement`1 servicecfg, Uri homeRealm, ClientCredentials userCredentials, ClientCredentials deviceCredentials, Int32 depthLevel, Boolean tryNetworkCred)
   at Microsoft.Xrm.Tooling.Connector.CrmWebSvc.CreateAndAuthenticateProxy[T](IServiceManagement`1 servicecfg, Uri ServiceUri, Uri homeRealm, ClientCredentials userCredentials, ClientCredentials deviceCredentials, String LogString)
   at Microsoft.Xrm.Tooling.Connector.CrmWebSvc.DiscoverOrganizations(Uri discoveryServiceUri, Uri homeRealmUri, ClientCredentials clientCredentials, ClientCredentials deviceCredentials)
   at Microsoft.Xrm.Tooling.CrmConnectControl.CrmConnectionManager.ValidateServerConnection(CrmOrgByServer selectedOrg)

**Comment by MscrmTools** [comment]
It means your IFD deployment is not well configured. There are some article on Internet about this error message

**Comment by Roxville** [comment]
Actually, it is not an issue with IFD at all. 

I played with the Plugin Registration tool and could login in without issue AFTER I found an article referencing issues where domain names are non-standard (.local).  Our domain has been around for quite a while so is #####.local

To make the tool work I selected not to use IFD since the URL is my internal also on the authentication screen I had to enter:

Domain: #####.local
Username: administrator@######.local (without this it would fail)
Password: ######

With that configuration it works without any issues.  May be worth testing and documenting in your wiki.
